Resources/Locale/en-US/_NF/bluespace-events/events.ftl

```diff
@@ -1,8 +1,8 @@
-station-event-bluespace-vault-start-announcement = A NanoTrasen autonomous armored vault transport has made an FTL error and will soon arrive nearby. Your sector will be rewarded for it's safe return.
-station-event-bluespace-vault-end-announcement = We have successfully retrieved the vault from your sector and have reimbursed your nearby Frontier Outpost accordingly.
+station-event-bluespace-vault-start-announcement = A NanoTrasen armored vault is inbound to the sector. The area 300m around the vault is a No Fly Zone. Station accounts will be rewarded for its safe return.
+station-event-bluespace-vault-end-announcement = We have retrieved the vault from your sector. Station accounts have been remunerated.
 
-station-event-bluespace-cache-start-announcement = A Syndicate transport has been intercepted from FTL and will soon arrive nearby. Guard the armored weapons cache until NanoTrasen can retrieve it for a reward.
-station-event-bluespace-cache-end-announcement = We have successfully retrieved the Syndicate weapons cache from your sector and have reimbursed your nearby Frontier Outpost accordingly.
+station-event-bluespace-cache-start-announcement = A Syndicate weapons cache is inbound to the sector. The area 300m around the cache is a No Fly Zone. Station accounts will be rewarded for guarding the cache.
```

Comment:
```suggestion
station-event-bluespace-cache-start-announcement = A Syndicate weapons cache is inbound to the sector. The NFSD may declare the cache a No Fly Zone. Station accounts will be rewarded for guarding the cache.
```

Reply:
Merging these suggestions - if it turns out a fixed range makes sense in play, happy to revert.